packages/react-native/ReactAndroid/src/test/java/com/facebook/react/devsupport/JSDebuggerWebSocketClientTest.kt

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.devsupport
+
+import com.facebook.react.common.JavascriptException
+import com.facebook.react.devsupport.JSDebuggerWebSocketClient.JSDebuggerCallback
+import okhttp3.WebSocket
+import okio.ByteString
+import okio.ByteString.Companion.encodeUtf8
+import org.junit.Ignore
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.powermock.api.mockito.PowerMockito
+import org.powermock.core.classloader.annotations.PowerMockIgnore
+import org.powermock.core.classloader.annotations.PrepareForTest
+import org.powermock.modules.junit4.rule.PowerMockRule
+import org.robolectric.RobolectricTestRunner
+
+@PrepareForTest(JSDebuggerWebSocketClient::class)
+@RunWith(RobolectricTestRunner::class)
+@PowerMockIgnore(
+    "org.mockito.*",
+    "org.robolectric.*",
+    "androidx.*",
+    "android.*"
+)
+
+class JSDebuggerWebSocketClientTest {
+
+    @JvmField
+    @Rule
+    val rule = PowerMockRule()
+
+    @Test
+    @Throws(Exception::class)
+    fun test_prepareJSRuntime_ShouldSendCorrectMessage() {
+        val cb = PowerMockito.mock(JSDebuggerCallback::class.java)
+        val client = PowerMockito.spy(JSDebuggerWebSocketClient())
+        client.prepareJSRuntime(cb)
+        PowerMockito.verifyPrivate(client)
+            .invoke("sendMessage", 0, "{\"id\":0,\"method\":\"prepareJSRuntime\"}")
+    }
+
+    @Test
+    @Throws(Exception::class)
```

Comment:
I think all those `@Throws(Exception::class)` can go, can you verify?

Reply:
Removed all occurrences of `@Throws(Exception::class` and verified they were optional.